panel/src/components/Views/FileView.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <k-inside>
+  <k-inside :lock="lock">
```

Comment:
Same here. I'm not sure I fully understand how locking is passed down.

Reply:
The view receives the `lock` prop (defined in ModelView) from Inertia, passes it down to Inside which passes it down to FormButtons.

Reply:
Ok, that makes sense, but somehow the form button component does not receive the lock prop in inside.